index.html

```diff
@@ -1,11 +1,151 @@
 <!DOCTYPE html>
 <html lang="en">
   <head>
-    <meta charset="UTF-8">
-    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta charset="UTF-8" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Homework 1</title>
   </head>
   <body>
-    <h1>Start developing here</h1>
+    <header class="header">
+      <a class="header__logo logo" href="#">WebStudio</a>
+
+      <nav class="header__nav nav">
+        <ul class="nav__list">
+          <li class="nav__item active"><a href="#">Студія</a></li>
+          <li class="nav__item"><a href="#">Портфоліо</a></li>
+          <li class="nav__item"><a href="#">Контакти</a></li>
+        </ul>
+      </nav>
+
+      <a class="header__email email active" href="#">info@devstudio.com</a>
+      <p class="header__number namber">+38 096 111 11 11</p>
+    </header>
+    <main class="main">
+      <section class="main__banner banner">
+        <h1 class="banner__main-text">Ефективні рішення для вашого бізнесу</h1>
+        <button class="banner__button">Замовити послугу</button>
+      </section>
+
+      <section class="main__features features">
+        <article class="features__item item">
```

Comment:
better name for this block (and the rest would be `feature`
make sure to make similar elements as a `list`
```
<ul class="features__list">
  <li class="features__item feature>
    <h3 class="feature__title">...</h3>
    <p class="feature__text> ... </p>
  </li>
  ...
</ul>
```